libnmstate/schemas/operational-state.yaml

```diff
@@ -177,6 +178,33 @@ definitions:
           required:
             - id
             - base-iface
+  interface-vxlan:
+    rw:
+      properties:
+        type:
+          type: string
+          enum:
+            - vxlan
+        vxlan:
+          type: object
+          properties:
+            id:
+              type: integer
+              minimum: 0
+              maximum: 16777216
+            remote:
+              type: string
+            destination-port:
+              type: integer
+            parent:
+              type: string
+          required:
+            - id
+            - parent
+            - remote
```

Comment:
`remote` should be optional.

Reply:
libnm fails if we don't pass it, looks like is mandatory at NetworkManager